### ISSUE TITLE
Widgets

### ISSUE THREAD

**Issue by LeonNicholls** [LeonNicholls/Open-Launcher-for-GTV#2]
Support launcher widgets

**Comment by arpu** [comment]
you working on this?

**Comment by LeonNicholls** [comment]
Yes. There are some code already checked in but not activated. Supporting widgets is a large effort since the design is different from mobile devices.

**Comment by arpu** [comment]
hello
i want to use this launcher on my "normal" android tv box (ist not gtv)
do i need to fork your launcher or is it possible to integrate this in this launcher? 

best regards

**Comment by LeonNicholls** [comment]
Touch events and gestures will also be supported but it will be optimized for d-pad/arrow navigation for Google TV.
